### ISSUE TITLE
DST Staging Review: Visual focus indicator has insufficient color contrast against adjacent (background) color

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#4683]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** New Tabs Design VADS Component
**Staging Review ticket:** [Staging review: New Tabs Design VADS Component #115098](https://github.com/department-of-veterans-affairs/va.gov-team/issues/115098)

## Findings details
**VA.gov Experience Standard - issue:** [User can identify an element and its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**This is an issue with the:** 
**High-priority:** No
**Collab Cycle Reviewer:** @jasonday  (Accessibility)

### Description

The yellow (#face00) visual focus indicator has a contrast ratio of 1.5:1, below the required 3:1 to meet WCAG 1.4.11

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="970" height="466" alt="visual focus indicator color contrast issue" src="https://github.com/user-attachments/assets/78b447de-bb70-46b7-912c-bc1f33ac0c25" />

</details>

## Recommended action
Update global visual focus indicators. See https://github.com/department-of-veterans-affairs/component-library/pull/1678#pullrequestreviw-3012253337 

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.4.11 Non-Text Contrast - Level AA](https://www.w3.org/TR/WCAG22/#non-text-contrast)
- **Modality:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for DST

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation
